lib/fluent/plugin/out_rdkafka2.rb

```diff
@@ -30,7 +30,10 @@ class Rdkafka::Producer
   def close(timeout = nil)
     rdkafka_version = Rdkafka::VERSION || '0.0.0'
     # Rdkafka version >= 0.12.0 changed its internals
-    if Gem::Version::create(rdkafka_version) >= Gem::Version.create('0.12.0')
+    # but reverted in >= 0.13.1
```

Comment:
Is the version `0.13.1` correct in this comment?

Reply:
I've overlooked the existence of 0.13.0.

Reply:
Fixed comment.